db/migrate/20230830115850_add_missing_indexes_and_fks.rb

```diff
@@ -0,0 +1,12 @@
+class AddMissingIndexesAndFks < ActiveRecord::Migration[6.1]
+  def change
+    add_index :timeline_events, :evaluator_id
+    add_index :timeline_events, :target_id
+    add_foreign_key :timeline_events, :targets
+
+    add_index :targets, :target_group_id
+    add_foreign_key :markdown_attachments, :schools
+    add_foreign_key :timeline_event_owners, :timeline_events
+
```

Comment:
The removal of FK relation from student to level is not removed as mentioned in the PR checklist.